topics/go/algorithms/strings/palindrome/palindrome.go

```diff
@@ -0,0 +1,21 @@
+package strings
+
+import strings "github.com/ardanlabs/gotraining/topics/go/algorithms/strings/reverse"
+
+func IsPalindrome(input string) bool {
```

Comment:
If you make this start with a captial I then you need a comment to pass `golint`. Make sure the linter doesn't fail or make this unexported.

Reply:
Done.